src/NuGet.Core/NuGet.Protocol.Core.v3/LocalRepositories/LocalV2FindPackageByIdResource.cs

```diff
@@ -106,7 +106,7 @@ private List<CachedPackageInfo> GetPackageInfos(string id)
                         throw new FatalProtocolException(message, ex);
                     }
 
-                    if (string.Equals(reader.GetId(), id, StringComparison.Ordinal))
+                    if (string.Equals(reader.GetId(), id, StringComparison.OrdinalIgnoreCase))
```

Comment:
Any reason `string.Equals` is used here rather than `StringComparer`? Are they interchangeable?